keel-veto/src/main/kotlin/com/netflix/spinnaker/keel/veto/VetoController.kt

```diff
@@ -78,12 +78,18 @@ class VetoController(
     path = ["/{name}"],
     produces = [MediaType.APPLICATION_JSON_VALUE]
   )
-  fun passMessage(@PathVariable name: String, @RequestBody message: Map<String, Any>) =
-    vetos.forEach { veto ->
-      if (veto.name().equals(name, ignoreCase = true)) {
-        veto.passMessage(message)
+  fun passMessage(@PathVariable name: String, @RequestBody message: Map<String, Any>) {
+    val veto = vetos.find { it.name().equals(name, true) }
+    if (veto == null) {
+      throw VetoNotFoundException(name)
+    } else {
+      vetos.forEach { veto ->
```

Comment:
something seems a little odd to me about looping over all the vetos looking for all the ones that match, specifically in the context of the rest of this controller acting like there's only ever one matching veto name. any objections to making this act more like the other methods (i.e. use a single `veto` variable to pass the message to)?

Reply:
That's definitely better. There should be only one.

You can do:
```
val veto = vetos.find { it.name().equals(name, true) } ?: throw VetoNotFoundException(name)
veto.passMessage(message)
```

Reply:
ooo that elvis business is great, i might just swap all these methods over to use that syntax while i'm in here 👀

Reply:
I was definitely hoping you'd do that! I'm not great at idiomatic kotlin, but getting better.

Reply:
3f6bd16